x-pack/test/cases_api_integration/security_and_spaces/tests/common/cases/find_cases.ts

```diff
@@ -663,45 +715,6 @@ export default ({ getService }: FtrProviderContext): void => {
         });
       }
 
-      it('should return the correct cases when trying to exploit RBAC through the search query parameter', async () => {
```

Comment:
This is no longer applicable because you can't search the `owner` field and that is covered by the tests I added above.